common/helpers/dateHelperUTCFormatted.js

```diff
@@ -54,6 +58,25 @@ function rangeShort(start, end, tc) {
   return result
 }
 
+// long end part of dateTime range format
+// doesn't repeat end date if on the same day
+function rangeLongEnd(start, end, tc) {
```

Comment:
Can you add tests for this?

Reply:
We should write tests for any code that could realistically break in the future. If you have time, sure. If you only have time for that one function, also okay. I don't think we necessarily need tests for `hourLong` but that might be up to personal preference.